### ISSUE TITLE
New RSM424 pcb version

### ISSUE THREAD

**Issue by Mika-1818** [ramapongithub/RS41-UBLOX-GPS#1]
There is a new version of the RS41 in use now. It has a new GPS chip and a overall new layout of the pcb. The serial number I think starts with W.

**Comment by ra1nb0w** [comment]
Hi, have you tried to cut the board with UBX-M10050? thanks

**Comment by ramapongithub** [comment]
> Hi, have you tried to cut the board with UBX-M10050? thanks

No, but SvenEDDB in Telegram-group "Radiosonden Sondentechnik" has done it. I asked him for his pictures.
For better reception. he keeps more of the PCB to keep the complete antenna.

<img width="1242" height="1280" alt="Image" src="https://github.com/user-attachments/assets/4defff45-8894-4df8-ac05-4bfd2f8bb0bc" />

<img width="1280" height="1272" alt="Image" src="https://github.com/user-attachments/assets/1ecd65c8-a38c-4b81-92ae-65e6bd94f6b8" />

<img width="1262" height="1280" alt="Image" src="https://github.com/user-attachments/assets/74f0aa62-3334-49db-a2fb-0bcb57995e54" />

<img width="1137" height="1280" alt="Image" src="https://github.com/user-attachments/assets/89a3ae65-4db5-4962-ad74-25143147d32b" />

**Comment by ra1nb0w** [comment]
thank you very much @ramapongithub
